free5gc-operator/package-context.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: v1
 kind: ConfigMap
+
 metadata:
   name: kptfile.kpt.dev
   annotations:
     config.kubernetes.io/local-config: "true"
+
 data:
-  name: example
+  name: nehpio
```

Comment:
This should stay "example". It's an auto-generated file though, so it will get overwritten anyway.

"name' in package-context.yaml is set by kpt / Porch to the package name when you `kpt pkg get --for-deployment` or you clone to a ``Deployment` repository.